packages/cerebral/src/operators/debounce.js

```diff
@@ -1,3 +1,3 @@
-import debounce from 'function-tree/lib/factories/debounce'
+import debounce from 'function-tree/factories/debounce'
```

Comment:
Yes it will. Moreover tests run with `lerna run test` builds all packages before running tests.
And  I can't see how 'function-tree/factories/debounce' path is worked for you.